### DISCUSSION TITLE
Loom File Upload to Scope

### DISCUSSION THREAD

**Discussion by Su870** [aertslab/SCENIC#381] [* General SCENIC questions]
Hello,

I have created my final loom file after running AuCell using the code below.
!pyscenic aucell \
    {f_loom_path_scenic} \
    reg.csv \
    --output {f_pyscenic_output} \
    --num_workers 20

And then I visualized my AUC matrix with these lines of code:
import json
import zlib
import base64

----collect SCENIC AUCell output
lf = lp.connect( f_pyscenic_output, mode='r+', validate=False )
auc_mtx = pd.DataFrame( lf.ca.RegulonsAUC, index=lf.ca.CellID)
lf.close()
import umap

-----UMAP
runUmap = umap.UMAP(n_neighbors=10, min_dist=0.4, metric='correlation').fit_transform
dr_umap = runUmap( auc_mtx )
pd.DataFrame(dr_umap, columns=['X', 'Y'], index=auc_mtx.index).to_csv( "scenic_umap.txt", sep='\t')
-----tSNE
tsne = TSNE( n_jobs=20 )
dr_tsne = tsne.fit_transform( auc_mtx )
pd.DataFrame(dr_tsne, columns=['X', 'Y'], index=auc_mtx.index).to_csv( "scenic_tsne.txt", sep='\t')

The loom file created from AUCell has contents in the loom but when I upload it to SCope, it is blank. I only know the loom file is not empty because from my file explorer it says size is 22,136 KB.
![image](https://user-images.githubusercontent.com/114273669/217663788-816309a3-fece-4ba6-b79f-17ff902f859c.png)

I did not integrate SCENIC with Scanpy because I do not need it, could that be the problem?

I have two questions:
How do I see my loom results on SCope?
What can I do with the umap and tsne files that I made?

I am new to pyscenic so forgive the simple questions!
Thank you

**Comment by ybaeus** [comment]
Were you able to solve it? i am facing the same problem
